db/data-generator.js

```diff
@@ -0,0 +1,89 @@
+const faker = require('faker');
+const fs = require('fs');
+const moment = require('moment');
+
+const generateRandomNumber = function(min, max, decimal) {
```

Comment:
Faker has 'faker.random.number' and 'faker.random.amount' that might be helpful here in case you were looking for a simpler approach